app/controllers/socure_webhook_controller.rb

```diff
@@ -41,4 +45,21 @@ def verify_queue(authorization_header:)
       )
     end
   end
+
+  def log_webhook_receipt
+    event = socure_params[:event]
+    return if event.blank?
+
```

Comment:
with the `socure_params[:event].blank?` in main method, this is a redundant check now?
```suggestion
    event = socure_params[:event]

```

Reply:
Good catch - thanks!